fastnode2vec/node2vec.py

```diff
@@ -14,38 +16,77 @@ def set_seed(seed):
 class Node2Vec(Word2Vec):
     def __init__(
         self,
-        graph,
-        dim,
-        walk_length,
-        context,
-        p=1.0,
-        q=1.0,
-        workers=1,
-        batch_walks=None,
-        seed=None,
-        **args,
+        graph: Graph,
+        dim: int,
+        walk_length: int,
+        window: int,
+        p: float = 1.0,
+        q: float = 1.0,
+        workers: Union[int, str] = "auto",
```

Comment:
Great! I'm not sure that I want to use all cpu as default. Also, I _think_ that -1 already uses all cpu so "auto" is not needed.

Could you remove that part?

Reply:
Sure, I'll document it a bit more to suggest the `-1` thing.

Reply:
I don't think it shoud be *suggested*. Multiple CPUs give a different result because the accumulation of gradients is not done in a thread-safe manner this is randomized by the OS thread scheduling. But documenting that -1 uses all cpus would be useful.

Reply:
I see that the default in GenSim is `3`, but I am not sure why. Do you prefer `1` as it was before or `3` as in gensim?